### PR TITLE
if load enable apikeys by config

### PR DESCRIPTION
**Pull request by wilhelmguo**

### CODE REVIEW

**Comment by wilhelmguo** [comment]
@chengyumeng EnableApiKeys default set true

**Comment by wilhelmguo** [comment]
refer https://github.com/Qihoo360/wayne/issues/9

File: src/frontend/src/app/portal/sidenav-namespace/sidenav-namespace.component.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
   title="APIKeys"
   routerLink="/portal/namespace/{{cacheService.currentNamespace?.id}}/apikey"
   routerLinkActive="active"
-  *ngIf="authService.currentNamespacePermission.apiKey.read || authService.currentUser.admin">
+  *ngIf="this.authService.config['enableApiKeys'] && (authService.currentNamespacePermission.apiKey.read || authService.currentUser.admin)">
```
Comment (BennieMeng) [inline src/frontend/src/app/portal/sidenav-namespace/sidenav-namespace.component.html]: 为什么要加this

**Pull request merged**